### DISCUSSION TITLE
Sharing Specific Pages in a Public Collection

### DISCUSSION THREAD

**Discussion by edwarddarrah** [outline/outline#4920] [General] [answered]
I am using Outline for our Public help docs. Is it possible to share specific pages in a collection? For example, I would like to direct a customer to a particular page to answer their support question.

**Comment by tommoor** [comment, marked as answer]
Yes, but the easiest way right now to get this link is to go to the publicly shared page, find the nested page you wish to share and copy the url from the browser address bar

**Reply by edwarddarrah** [reply]
Thank you, I did not think to do that. I just tested sharing a page section link, and that worked too.
